tools/eval/datasets/minisupervisely.py

```diff
@@ -0,0 +1,196 @@
+import os
+import cv2 as cv
+import numpy as np
+from tqdm import tqdm
+
+
+class MiniSupervisely : 
```

Comment:
Could you add a comment as reference to the original evaluation method in PaddleSeg? Something like this:

```.diff
class MiniSupervisely : 
+    '''
+       Refer to https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.7/paddleseg/core/val.py
+       for official evaluation implementation.
+    '''
```